print(1)
try:
	print(2)
	print(i)
	print(3)
except (NameError):
	print('except')
print(4)